Classes/Issues/IssuesViewController.swift

```diff
@@ -529,12 +523,13 @@ FlatCacheListener {
             isRoot: false
             )
             else { return }
-        sentComments.append(comment)
 
-        let collectionView = feed.collectionView
-        feed.adapter.performUpdates(animated: false, completion: { _ in
-            collectionView.slk_scrollToBottom(animated: true)
-        })
+        needsScrollToBottom = true
+
+        let newResult = previous.updated(
+            timelinePages: previous.timelinePages(appending: [comment])
```

Comment:
This (the `timelinePages(appending:)`) looked really weird and had to look up what it was actually doing. Can this be renamed to more clearly state the side effect?

At least we can add a similar method that accepts just one value instead of having to wrap it in an array.

Reply:
What would you rename it to? FWIW there is no side effect, it returns new pages appending the array you give it.